### ISSUE TITLE
(Meta) Target state of MeltanoHub for Meltano 2.0 Launch

### ISSUE THREAD

**Issue by tayloramurphy** [meltano/hub#517]
- [ ] All taps and targets are listed as Extractors and Loaders - #254 
- [ ] Maintain automated index of Singer plugins pages - likely wouldn't link on main page - #254 
- [ ] Search bar on the main page
  - [AJ's update] On Redesign site but not yet on hub.meltano.com: https://gitlab.com/meltano/hub/-/issues/197 or an
- [ ] Convert existing markdown file to a “usage.md”
  - [AJ's update] Logged new issue here for tracking: #253
- [ ] Some indication about support level based on capabilities / settings w/ link to some doc that has that
  - [AJ's update] As with the above, could be handled as part of: #253
- [ ] YAML File + API + UI updates via #236 
- [ ] Split Utilities into Quality and Analysis Categories #256 
- [ ] Distinguish standalone files bundles #255 
- [ ] Update repo links once migration is complete #257
